src/ex2/main.js

```diff
@@ -0,0 +1,59 @@
+import  ItemManager  from "./ItemManager.js";
+class Main {
+  constructor() {
+   this.itemManager = new ItemManager();
+   this.addButton = document.getElementById("add-btn");
+   this.inputTaskName = document.getElementById("input-task");
+   this.sortButton = document.getElementById("sort-btn");
+   this.clearAllButton = document.getElementById("clear-btn");
+  }
+
+  init() {
+    this.ListenToAddItem()
+    this.listenToSortButton();
+    this.listenToClearAllTasks();
+
+   
+
+  }
+ 
+   ListenToAddItem(){
```

Comment:
stick to the conventions please
(fix to camel-case everywhere)

```suggestion
   listenToAddItem(){
```